utils/promptUtils.test.js

```diff
@@ -0,0 +1,95 @@
+const {autoComplete, confirmUserAction} = require('./promptUtils');
+const prompt = require('prompt-sync')();
+
+describe('promptUtils', () => {
+    describe('autoComplete', () => {
+        it('should return a function', () => {
```

Comment:
NIT:
Please confirm if the indentation is correct, it seems like 4 spaces, but in line 62 seems like 2 spaces

Reply:
Will check, thanks!

Reply:
I formatted the code, it should fix the indentation. thanks!